Danggn/Screen/Home/Controller/ListItemViewController.swift

```diff
@@ -8,10 +8,105 @@
 import UIKit
 
 class ListItemViewController: UIViewController {
-
+    
+    @IBOutlet weak var listCollectionView: UICollectionView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        setNavigationUI()
+        registerCell()
+    }
+    
+    @IBAction func itemCreateButtondidTap(_ sender: Any) {
+        
+    }
+}
+
+extension ListItemViewController {
+    func setNavigationUI() {
+        
+        let navigationBar = UINavigationBarAppearance()
```

Comment:
```suggestion
        let navigationBarAppearance = UINavigationBarAppearance()
```
navigationBarApperance가 더 좋은 네이밍이지 않을까 싶습니다!

30번 라인에서 
```swift
self.navigationController?.navigationBar.standardAppearance = navigationBarAppearance
```
로 되어있는게 혼동의 여지없이 더 잘 읽히는 것 같습닡다!!😀

Reply:
좋은것 같습니다! 해당사항 반영해서 수정하겠습니다! 감사합니다~!